packages/app/src/components/applicationStages/modals/MarkStageAsCompleteModal.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+
+Modal.setAppElement('#root');
+
+const MarkStageAsCompleteModal = ({ advanceStage, currentStatus }) => {
+    const [isOpen, setIsOpen] = useState(false)
+    
+    function openModal() {
+      setIsOpen(true);
+    };
+
+    function closeModal() {
+      setIsOpen(false);
+    }   
+
+  return (
+    <>
```

Comment:
Is there a reason for this non-defined element?

Reply:
I'm not sure why I didn't define it. I'll correct it.